Data-Collection-Inventory/2021Q2_csvs/data_collection_project_specimen_type_classification.csv

```diff
@@ -0,0 +1,80 @@
+project_reference_id,specimen_type_name
+dong_antero_proj,brain section set
+dong_retro_proj,brain section set
+dong_trio_proj,brain section set
+dulac_pag_sn_10x_proj,cell nucleus
+ecker_hu_DNAseq_proj,tissue sample
+ecker_hu_sn_ATACseq_proj,cell nucleus
+ecker_hu_sn_mCseq_proj,cell nucleus
+ecker_sn_ATACseq_proj,cell nucleus
+ecker_sn_mCseq_proj,cell nucleus
+ecker_sn_mCseq_retro_proj,cell nucleus
+ecker_trio_proj,brain section set
+feng_fish_proj,brain section set
+feng_fish_proj,cell in slice
+feng_sn_dropseq_proj,cell nucleus
+fischl_mri_proj,brain hemisphere
+fischl_mri_proj,brain slice
+fischl_mri_proj,tissue sample
+huang_antero_proj,brain section set
+huang_fmost_proj,whole brain
+huang_fmost_proj,whole cell
+huang_input_conn_proj,whole brain
+huang_pn_proj,bulk nucleus
+huang_pn_proj,cell nucleus
+huang_retro_proj,brain section set
+huang_sn_10xv2_proj,cell nucleus
+huang_stpt_conn_proj,whole brain
+huang_stpt_count_proj,whole brain
+HumanM110xProj,cell nucleus
+human_cortex,cell nucleus
+kriegstein_sc_10x_proj,whole cell
+kriegstein_sc_ATACseq_proj,cell nucleus
+kriegstein_sn_10x_proj,cell nucleus
+lein_sn_10x,cell nucleus
+lein_sn_SSv4_proj,cell nucleus
+mueller_mouselight_proj,whole brain
+nowakowski_evobc_proj,whole cell
+nowakowski_pndev_10xv3_proj,cell nucleus
+osten_stpt_proj,whole brain
+regev_cisi_proj,brain slice
+tasic_ATACseq_proj,whole cell
+tasic_stpt_proj,whole brain
+tilgner_sc_10x_PB,cell nucleus
+TransgenicSTPT,whole brain
+yang_MORF_confocal_proj,brain section set
+yang_MORF_confocal_proj,whole cell
+zeng_anderson_sc_10xv2_proj,whole cell
+zeng_fmost_proj,whole brain
+zeng_fmost_proj,whole cell
+zeng_mouselight_proj,whole brain
+zeng_retro_proj,whole brain
+zeng_sc_10x_proj,whole cell
+zeng_sc_SSv4_proj,whole cell
+zeng_sn_10x_proj,cell nucleus
+zeng_sn_SSv4_proj,cell nucleus
+zeng_tolias_pseq_proj,cell in slice
+zeng_tolias_pseq_proj,cell nucleus
+zhangK_snareseq,cell nucleus
+zhangl_anatomy_proj,spinal cord slice
```

Comment:
this is whole spinal cord, not slice

Reply:
Changed `zhangl_anatomy_proj` specimen type to (whole) `spinal cord`.